### DISCUSSION TITLE
Creating releases

### DISCUSSION THREAD

**Discussion by trhgquan** [khongsomeo/GPA-OOP#62] [Ideas]
Releases should be created automatically during a workflow:
- Build on multiple OS.

**Comment by hwangswan** [comment]
Testing on https://github.com/khongsomeo/demo-cpp-multibuild, coming soon!

**Comment by hwangswan** [comment]
The idea is to create a new workflow for releases, working only with pushes.

**Comment by trhgquan** [comment]
https://kevsoft.net/2020/06/10/running-github-action-steps-and-jobs-only-on-push-to-master.html

We can split release to a separate job, then using conditional statement we can make it handle pushes only.
